tests/composer/composer-symfony6-min.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "propel/propel",
+    "type": "library",
+    "description": "Propel2 is an open-source Object-Relational Mapping (ORM) for PHP 5.5 and up.",
+    "keywords": [
+        "ORM",
+        "persistence",
+        "Active Record"
+    ],
+    "homepage": "http://www.propelorm.org/",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "William Durand",
+            "email": "william.durand1@gmail.com"
+        }
+    ],
+    "require": {
+        "php": ">=8.0",
+        "symfony/yaml": "~6.0.0",
+        "symfony/config": "~6.0.0",
+        "symfony/console": "~6.0.0",
+        "symfony/filesystem": "~6.0.0",
+        "symfony/finder": "~6.0.0",
+        "symfony/translation": "~6.0.0",
+        "symfony/validator": "~6.0.0",
+        "symfony/polyfill-php73": "^1.8",
+        "psr/log": "^1.0"
+    },
+    "require-dev": {
+        "ext-pdo": "*",
+        "ext-json": "*",
+        "monolog/monolog": "^1.3",
+        "phpstan/phpstan": "^0.12.4",
+        "phpunit/phpunit": "^8.0.0||^9.0.0",
```

Comment:
It is also unlikely that v8 would even install here, as we are using v9 only now right?

Reply:
Will check against main composer.json file.